pkg/processor/runtime/python/py/_nuclio_async_wrapper.py

```diff
@@ -140,6 +142,11 @@ async def initialize(self):
     async def _process_connection(self, sock):
         """Process all events for a single connection."""
         # signal start
```

Comment:
move "singal start" comment below, add a comment instead that you initialize a connection-specific logger for event handling.